app/js/services/preprocessing.js

```diff
@@ -35,14 +35,38 @@ angular
                 return formatTeamDisplayFilter(teamMap[element.relationships.team.data.id], false);
             }
         },
-				prettifyDate: function(date) {
-					if (date == undefined) { return; };
-					var monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
-					var year = parseInt(date.substring(0, 4));
-					var month = parseInt(date.substring(5, 7));
-					var day = parseInt(date.substring(8, 10));
-					return monthNames[month - 1] + " " + day + ", "+ year;
-				},
+        convertTimeAtrtributes: function(element, ...attributeArray){
```

Comment:
rmb to update spelling for all of the places u r using `convertTimeAtrtributes` -> `convertTimeAttributes`